components/automate-chef-io/content/docs/infra-server.md

```diff
@@ -148,3 +148,66 @@ Installations require elevated privileges, so run the commands as the superuser
     ```shell
       sudo chef-automate deploy config.toml
     ```
+
```

Comment:
I added the new section instead of pointing to the existing docs because I think having the content in one place is a better experience for someone new to Automate-deployed Infra Server; I'm fine with someone disagreeing and replacing the new section with pointers to the existing docs.